__tests__/src/core/coords.spec.js

```diff
@@ -0,0 +1,37 @@
+import Coords from '../../../src/core/coords';
+
+describe('Game', () => {
+    describe('Coords', () => {
+        it('should construct coords with default value', () => {
```

Comment:
If you're using jest for tests, use `test` instead of `it`. Otherwise you won't be able to filter tests correctly with their runner cli